app/javascript/dashboard/components/widgets/WootWriter/FullEditor.vue

```diff
@@ -173,6 +172,13 @@ export default {
           blur: () => {
             this.onBlur();
           },
+          paste: (view, event) => {
+            const data = event.clipboardData.files;
+            if (data.length > 0) {
+              data.forEach(file => this.uploadImageToStorage(file));
```

Comment:
Should we check if the pasted file is an image? Otherwise looks fine to me.

Reply:
Good suggestion. I have updated the code.